spec/yaml/swiftnav/sbp/integrity.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2018-2022 Swift Navigation Inc.
+# Contact: https://support.swiftnav.com
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.integrity
+description: Integrity flag messages
+stable: False
+public: False
+include:
+  - types.yaml
+  - gnss.yaml
+definitions:
+  - MSG_SSR_FLAG_HIGH_LEVEL:
+      id: 0x0BB9
+      short_desc: High level integrity flags
+      public: False
+      fields:
+        - stub:
```

Comment:
What's the plan for all these stubs? Are they going to be included in a release and then redefined later on? Or are they going to be defined properly on this PR before merging?

Reply:
It's part of a new proposal.
https://swift-nav.atlassian.net/wiki/spaces/~jason/pages/2198831192/Proposal+for+SBP+message+staging
In the `staging` branch the full definition of the message is going to be defined. While in `master` this stubbed version reserve the ids. Once the messages are mature enough and don't need to have any fields updated, they will be promoted from `staging` to `master`.